media-api/app/lib/elasticsearch/IsQueryFilter.scala

```diff
@@ -62,3 +66,26 @@ case class IsDeleted(isDeleted: Boolean) extends IsQueryFilter {
     (filters.existsOrMissing("softDeletedMetadata", _))(isDeleted)
   )
 }
+
+case class IsReapable(persistedRootCollections: List[String], persistenceIdentifier: String) extends IsQueryFilter {
+  val moreThanTwentyDaysOld = filters.date("uploadTime", Some(new DateTime(0)), Some(DateTime.now().minusDays(20))).getOrElse(matchAllQuery())
```

Comment:
you can make the from be `None` to get an open-ended search 
```suggestion
  val moreThanTwentyDaysOld = filters.date("uploadTime", None, Some(DateTime.now().minusDays(20))).getOrElse(matchAllQuery())
```